keycloak-rest-api-extensions/src/main/java/io/cloudtrust/keycloak/email/EmailSender.java

```diff
@@ -15,6 +18,17 @@
 
 public class EmailSender {
     public static Response sendMail(KeycloakSession session, RealmModel realm, EmailModel emailModel, Locale locale, Map<String, Object> attributes) {
+        String receiverAddress = emailModel.getRecipient();
+
+        EventBuilder eventBuilder = new EventBuilder(realm, session, session.getContext().getConnection());
+        eventBuilder.event(EventType.CUSTOM_REQUIRED_ACTION)
+                .user(session.users().getUserByEmail(realm, receiverAddress))
+                .detail("receiverAddress", receiverAddress)
+                .detail("themeRealmName", emailModel.getTheming().getThemeRealmName())
```

Comment:
In line 32, we check that getTheming is not null so I think we could have a null pointer here

Reply:
You're right, thanks!